client/trino-jdbc/src/main/java/io/trino/jdbc/TrinoDatabaseMetaData.java

```diff
@@ -83,7 +83,10 @@ public String getURL()
     public String getUserName()
             throws SQLException
     {
-        return connection.getUser();
+        try (ResultSet rs = select("SELECT current_user")) {
```

Comment:
Do we have tests for this `getUserName` method?

Reply:
I've added one.